src/app/modules/ix-forms/components/ix-list/ix-list-item/ix-list-item.harness.ts

```diff
@@ -27,7 +27,14 @@ export class IxListItemHarness extends ComponentHarness {
   }
 
   async fillForm(values: Record<string, unknown>): Promise<void> {
-    const controls = await this.getControlHarnessesDict();
-    return fillControlValues(controls, values);
+    const labels = Object.keys(values);
+    for (const label of labels) {
+      const controlsDict = await this.getControlHarnessesDict();
+      const control = controlsDict[label] as IxFormControlHarness;
```

Comment:
Maybe introduce a `fillControl` helper?

Reply:
`getControlHarnessesDict` is a situation specific method. Creating a helper method would require passing the method in as a param to be called. We can create class member methods but that's not reusable anywhere else so I think this is good enough.